### ISSUE TITLE
Are `bandgap_tol` or `bandgap_override` used anywhere?

### ISSUE THREAD

**Issue by Andrew-S-Rosen** [materialsproject/atomate2#552]
https://github.com/search?q=repo%3Amaterialsproject%2Fatomate2%20bandgap_tol&type=code

A quick search suggests they are class variables but aren't used anywhere. CC @janosh, @esoteric-ephemera.

**Comment by janosh** [comment]
Good catch! `bandgap_tol` and `bandgap_override` were used at one point in #504 when we still had the `set_kspacing` utility function in `src/atomate2/vasp/sets/mp.py` but that was removed in favor of `_set_kspacing` in `src/atomate2/vasp/sets/base.py` and I forgot to add the plumbing to pipe the parameters there.

I think we can remove `bandgap_override` since I don't see any current use cases for it. I vote in favor of keeping `bandgap_tol` and even making it a global `SETTINGS` field in `src/atomate2/settings.py`.
